backend/models.py

```diff
@@ -191,6 +191,6 @@ def serialize(self):
 """
 class ItemTemplate(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(50), nullable=False)
+    name = db.Column(db.String(50), nullable=True)
```

Comment:
This was causing an error with the database, which was what was causing init_db not to run this part or something. In the above code snippet, we don't define a name for the item template, and I don't think a name is necessary